content/en/ecosystem/distributions.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Distributions
+description:
+  Libraries, services, and apps with first-party support for OpenTelemetry.
+aliases: [/distributions]
+---
+
+OpenTelemetry [distributions](/docs/concepts/distributions/) are a way of
+customizing OpenTelemetry components so that they're easier to deploy and use
+with specific observability back ends.
+
+Any third-party can customize OpenTelemetry components with back-end, vendor, or
+end-user specific changes. The following list contains a sample of OpenTelemetry
+distributions and their type.
+
```

Comment:
Add a sentence here that calls out specifically, that a distribution is not a "must have", and that many organisations and vendors just live fine without one (especially for language SDKs, collector might be the exception here)

Reply:
How about this? https://github.com/open-telemetry/opentelemetry.io/pull/3854/commits/9dff456af19688f51b2fbff9f45323a94624a239